core/src/test/scala/org/apache/spark/sql/delta/DeltaHistoryManagerSuite.scala

```diff
@@ -47,6 +48,11 @@ trait DeltaTimeTravelTests extends QueryTest
     duration.toMillis
   }
 
+  protected override def sparkConf = {
```

Comment:
The change in this file is not needed since the test suite extends `DeltaSQLCommandTest`.

Reply:
I have reverted the changes. My local upstream/master was didn't have the latest version at the time I made this change. I have now reverted the changes.